python/rikai/mixin.py

```diff
@@ -62,12 +64,4 @@ def __eq__(self, o: object) -> bool:
 
     def open(self, mode="rb") -> BinaryIO:
         """Open the asset and returned as random-accessible file object."""
```

Comment:
Do you want to provide a note saying that the caller is responsible for closing the returned file object?